src/config/chain.ts

```diff
@@ -0,0 +1,17 @@
+import { endpointKey } from 'src/config/chainEndpoints';
+
+export enum astarChain {
+  SHIBUYA = 'Shibuya Testnet',
+  SHIDEN = 'Shiden',
+  ASTAR = 'Astar',
+  DEVELOPMENT = 'Development',
+}
+
+export type ASTAR_CHAIN = astarChain.SHIDEN | astarChain.ASTAR | astarChain.SHIBUYA;
+
+export type ASTAR_NETWORK_IDX = endpointKey.ASTAR | endpointKey.SHIDEN | endpointKey.SHIBUYA;
+
+export type ASTAR_EVM_NETWORK_IDX = 592 | 336 | 81;
```

Comment:
Can you check if we can get this form a node.

Reply:
This type is used for type annotation in [this variable](https://github.com/AstarNetwork/astar-apps/blob/main/src/hooks/useNetworkInfo.ts#L27) only. I don't think it is necessary to fetch it from node.